src/Controllers/CourseUserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Uasoft\Badaso\Module\LMSModule\Controllers;
+
+use Exception;
+use Illuminate\Validation\ValidationException;
+use Uasoft\Badaso\Controllers\Controller;
+use Uasoft\Badaso\Helpers\ApiResponse;
+use Uasoft\Badaso\Module\LMSModule\Models\Course;
+use Uasoft\Badaso\Module\LMSModule\Models\CourseUser;
+use Uasoft\Badaso\Module\LMSModule\Models\User;
+
+class CourseUserController extends Controller
+{
+    public function view()
+    {
+        try {
+            $user = auth()->user();
+
+            $courseUsers = CourseUser::where(
```

Comment:
nit: `$courseUserIds`, since this variable does not contain the CourseUser object